app/api/v1/projects.rb

```diff
@@ -51,7 +51,7 @@ def load_projects(archiveds)
       current_time = params[:current_time] || Time.current
       since = params[:since].months.ago
 
-      iteration = Central::Support::IterationService.new(project, current_time: current_time) if project
+      iteration = Central::Support::IterationService.new(project, since: since, current_time: current_time) if project
```

Comment:
that `since` wasn't being used and I saw that the initializer accepts a `since` so I think someone forgot to put it in the call